File: Kontrolrum/ClientApp/app/components/Preparedness/preparedness.ts
export class Preparedness {
    id: number;
    vehicle: {
        id: number;
        name: string;
        vehicleType: {
            id: number;
            name: string;
            vehicles: any[];

        };
        status: {
            id: number;
            code: string;
            meaning: string;
            vehicles: any[];
        }
    };
    incident: {
        id: number;
        address: string;
        number: string;
        postNr: string;
        city: string;
        description: string;
        status: string;
        journals: any[];
        preparednesses: any[];
        creationTime: Date;
    };
    emergencyCode: {
        id: number;
        code: string;
        preparednesses: any[];
    };
    creationTime: Date;
    status: string;
    loggers: any[];
}

File: CoreModels/Preparedness.cs
using System;
using System.Collections.Generic;

namespace CoreModels
{
    public class Preparedness
    {

        public Preparedness()
        {
            Loggers = new List<Logger>();
        }
        public int ID { get; set; }
        public Vehicle Vehicle { get; set; }
        public Incident Incident { get; set; }
        public EmergencyCode EmergencyCode { get; set; }
        public DateTimeOffset CreationTime { get; set; }
        public List<Logger> Loggers { get; set; }
        public string Status { get; set; }
    }
}
